design/token.go

```diff
@@ -38,6 +38,7 @@ var _ = a.Resource("token", func() {
 		)
 		a.Params(func() {
 			a.Param("for", d.String, "The resource for which the external token is being fetched, example https://github.com/fabric8-services/fabric8-auth or https://api.starter-us-east-2.openshift.com")
+			a.Param("force_pull", d.Boolean, "Pull the user's details for the specific connected account, example, the user's updated github username would be fetched from github")
```

Comment:
Mention that if it's not set or "false" then the user profile will be pulled only if it's missing in Auth. Otherwise the profile stored in Auth will be used instead of updating it from the provider.

Reply:
ad55329

Done.

Reply:
Done in 06656cecf2653d2e3d7fdf6d064543083f9a0323